analysis/druid/src/shadowlands/ConvokeSpirits.tsx

```diff
@@ -227,9 +227,10 @@ class ConvokeSpirits extends Analyzer {
     );
   }
 
-  onConvoke(_: ApplyBuffEvent) {
+  onConvoke(event: ApplyBuffEvent) {
```

Comment:
did you mean to add convoke stuff to this PR?

Reply:
Yeah, I actually need the timestamp info in Convoke in order to easily add the flourish vs. no-flourish info in the new HotGraph. While I was there I added timestamp to Convoke's display because I had been intending to do that for a while.